src/bindable-property.js

```diff
@@ -70,6 +87,37 @@ export class BindableProperty {
     return undefined;
   }
 
+  _configureReflection(target) {
+    if (target[reflectionConfigured]) return;
+
+    let method = 'propertyChanged';
+    let onChanged = target.prototype[method];
+    let hasHandler = !!onChanged;
+
+    let alteredHandler;
+    if (hasHandler) { // avoid ternary to make it consisten with the rest ?
```

Comment:
I thought about this part some more...
It does something very similar to `createObserver` (same file).

Wouldn't it be more efficient and more clear to integrate `callReflection` inside `createObserver`?